src/ApiClient.php

```diff
@@ -249,4 +249,25 @@ private function getRandomQueryServer(): int
     {
         return rand(1, 2);
     }
+
+    public function stockSummary(string $symbol): array
+    {
+        $qs = $this->getRandomQueryServer();
+        $cookieJar = new CookieJar();
+
+        // Initialize session cookies
+        $initialUrl = 'https://fc.yahoo.com';
+        $modules = 'financialData,quoteType,defaultKeyStatistics,assetProfile,summaryDetail';
```

Comment:
Readability suggestion: Please move this down to the "fetch quotes" where it's actually used.

Reply:
make sense Done! :)